src/Symfony/Component/VarExporter/Tests/Fixtures/datetime.php

```diff
@@ -1,12 +1,10 @@
 <?php
 
 return \Symfony\Component\VarExporter\Internal\Configurator::pop(
-    \Symfony\Component\VarExporter\Internal\Registry::push([
-        \Symfony\Component\VarExporter\Internal\Registry::$reflectors[\DateTime::class] ?? \Symfony\Component\VarExporter\Internal\Registry::getClassReflector(\DateTime::class, true, true),
-    ], [
-        clone \Symfony\Component\VarExporter\Internal\Registry::$prototypes[\DateTime::class],
-    ], [
-    ]),
+    $r = \Symfony\Component\VarExporter\Internal\Registry::class,
+    $o = [
+        clone (($p =& $r::$prototypes)[\DateTime::class] ?? $r::p(\DateTime::class, true)),
```

Comment:
why not keeping `\Symfony\Component\VarExporter\Internal\Registry::$prototypes` ? Is there any benefit to making the class name a variable ?

Reply:
accessing a local var is slightly, and reduces the size of the export as a bonus

Reply:
I was talking about `$r`, not about `$p`

Reply:
Oh sorry. It's purely for compactness, without any performance impact.

Reply:
I removed the`$r` alias, it's not useful in the end.